easybuild/easyconfigs/i/imkl/imkl-11.3.1.150-iimpi-2016.00-GCC-4.9.3.eb

```diff
@@ -0,0 +1,38 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'imkl'
+version = '11.3.1.150'
+
+homepage = 'http://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'iimpi', 'version': '2016.00-GCC-4.9.3'}
+
+sources = ['l_mkl_%(version)s.tgz']
+checksums = ['b57ff502b5f97f2f783e4bbda7ce42b3']
+
+dontcreateinstalldir = 'True'
+
+#license file
+import os
+license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+
+interfaces = True
+
+moduleclass = 'numlib'
+
+postinstallcmds = [
+    # extract the examples
+    'tar xvzf %(installdir)s/mkl/examples/examples_cluster.tgz -C %(installdir)s/mkl/examples/',
+    'tar xvzf %(installdir)s/mkl/examples/examples_core_c.tgz -C %(installdir)s/mkl/examples/',
+    'tar xvzf %(installdir)s/mkl/examples/examples_core_f.tgz -C %(installdir)s/mkl/examples/',
+    'tar xvzf %(installdir)s/mkl/examples/examples_f95.tgz -C %(installdir)s/mkl/examples/',
+    'tar xvzf %(installdir)s/mkl/examples/examples_mic.tgz -C %(installdir)s/mkl/examples/'
```

Comment:
hmm, it would be better to handle this via the imkl easyblock (do-once-and-forget)? thoughts @jas02?